frontend/awx/main/AwxLogin.tsx

```diff
@@ -43,6 +44,9 @@ export function AwxLogin(props: { children: React.ReactNode }) {
           refreshActiveAwxUser?.();
           void mutate(() => true);
         }}
+        icon={<ProductIcon style={{ height: 64 }} />}
+        brand={process.env.BRAND}
```

Comment:
For those env variables do we need have a fallback value?

Reply:
should not